topgun/pipelines/lots-ata-time-2.yml

```diff
@@ -9,84 +9,212 @@ jobs:
   plan:
   - get: timer
     trigger: true
-  - task: exit
-    config:
-      platform: linux
-      image_resource:
-        type: registry-image
-        source: {repository: concourse/dev}
-      run:
-        path: /bin/sh
-        args:
-        - -c
-        - |
-          exit 2
+  - in_parallel:
```

Comment:
@ddadlani instead of making a new topgun test cause its heavy, I decided to just update the existing one to now assert that in_parallel tasks get distributed evenly.

Reply:
Sounds good